src/test/python/deep_qa_test/models/reading_comprehension/attention_sum_reader_test.py

```diff
@@ -35,5 +37,13 @@ def test_train_does_not_crash(self):
                 },
                 "embedding_size": 5,
         }
-        solver = get_model(AttentionSumReader, args)
-        solver.train()
+        model = get_model(AttentionSumReader, args)
+        model.train()
+
+        # load the model that we serialized
+        loaded_model = get_model(AttentionSumReader, args)
+        loaded_model.load_model()
```

Comment:
Great, thanks for writing this test, and I'm glad it actually works.  I'm surprised it wasn't more work to make this work right.  Will the model files actually get deleted after this test?